lib/ansiblelint/formatters/__init__.py

```diff
@@ -26,3 +26,22 @@ def format(self, match):
                                 "E" + match.rule.id,
                                 match.message,
                                 )
+
+
+try:
+    from ansible.color import stringc
+    ANSIBLE_VERSION = 1
+except ImportError:
+    from ansible.utils.color import stringc
+    ANSIBLE_VERSION = 2
+
+class ColoredFormatter(object):
```

Comment:
Rather than using a new formatter, perhaps update the existing formatters to handle colour if it's switched on.